test/unit/routes/test-reductions.js

```diff
@@ -14,34 +13,26 @@ const EDIT_REDUCTION_PAGE_URL = '/offender-manager/1/edit-reduction'
 const EDIT_REDUCTION_POST_URL = '/offender-manager/1/edit-reduction'
 const UPDATE_REDUCTION_STATUS_POST_URL = '/offender-manager/1/update-reduction-status'
 
-const addReduction = {
-  title: 'Title',
-  subTitle: 'SubTitle',
-  breadcrumbs: {},
-  subNav: {},
-  linkId: 1,
-  referenceData: {}
-}
-
 const getReductionNoTextResult = {
   title: 'Title',
   subTitle: 'SubTitle',
   breadcrumbs: {},
-  subNav: {}
+  subNav: []
 }
 
 const getReductionsSuccessTextResult = {
```

Comment:
This is the results from getAddReductionsRefData,  can you rename it and remove the successText?